From 30ae92c1678dcab79407fd2a03635d11141e666c Mon Sep 17 00:00:00 2001
From: chewitt <github@chrishewitt.net>
Date: Wed, 24 Apr 2019 10:52:06 +0000
Subject: [PATCH 36/37] dt-bindings: arm: amlogic: Add support for the
 SmartLabs SML-5442TW

The SmartLabs SML-5442TW is the next-gen/4K IPTV/DVB-S STB for O2 Czech based
on the Amlogic P231 reference design using the S905D chipset with widevine L1
license. Hardware specification:

- 2GB DDR3 RAM
- 8GB eMMC storage
- 10/100 Base-T Ethernet
- 802.11 a/b/g/n/ac + BT 4.1 HS sdio wireless module (QCA9377)
- 2x single colour and 1x dual colour LEDs on the front panel
- 1x reset button on the front panel
- HDMI 2.0 (4k@60p) video
- Composite video + 2-channel audio output on 3.5mm jack
- S/PDIF audio output
- Single DVB-S tuner (AVL6762/MxL608)
- 2x USB 2.0 ports
- 1x micro SD card slot
- UART pins (internal)

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.txt b/Documentation/devicetree/bindings/arm/amlogic.txt
index 7f40cb5f490b..2e706c6dd175 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.txt
+++ b/Documentation/devicetree/bindings/arm/amlogic.txt
@@ -92,6 +92,7 @@ Board compatible values (alphabetically, grouped by SoC):
   - "amlogic,p230" (Meson gxl s905d)
   - "amlogic,p231" (Meson gxl s905d)
   - "phicomm,n1" (Meson gxl s905d)
+  - "smartlabs,sml5442tw" (Meson gxl s905d)
 
   - "amlogic,p241" (Meson gxl s805x)
   - "libretech,aml-s805x-ac" (Meson gxl s805x)
-- 
2.17.1

